packages/stream_chat_flutter/lib/src/stream_chat_configuration.dart

```diff
@@ -0,0 +1,221 @@
+import 'package:flutter/material.dart';
+import 'package:stream_chat_flutter/stream_chat_flutter.dart';
+
+/// {@template streamChatConfigurationProvider}
+/// Inherited widget providing the [StreamChatConfiguration] to the widget tree
+/// {@endtemplate}
+class StreamChatConfigurationProvider extends InheritedWidget {
```

Comment:
I would rename this to `StreamChatConfiguration` and the data to `StreamChatConfigurationData`

Reply:
makes sense